### PR TITLE
Red border when product is sold out

### LINKED ISSUE #41: Show out of stock
Products should be marked out of stock when no stock remains

### PR DESCRIPTION
**Pull request by VandorpeDavid**
Resolves #41

### CODE REVIEW

File: app/assets/stylesheets/products.css.scss [hunk 1]
```diff
@@ -32,3 +32,7 @@
     vertical-align: middle;
   }
 }
+
+div.out-of-stock {
+  border-color:red;
```
Comment (Silox) [inline app/assets/stylesheets/products.css.scss]: space between name and value

File: app/assets/stylesheets/products.css.scss [hunk 2]
```diff
@@ -32,3 +32,7 @@
...
+}
```
Comment (Silox) [inline app/assets/stylesheets/products.css.scss]: add newline at end of file

File: app/views/order_items/_order_item.html.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <div class="col-md-3 form_products">
-  <div class="thumbnail ">
+  <div class="thumbnail<%= product.stock ==0 ? ' out-of-stock' : '' %>">
```
Comment (Silox) [inline app/views/order_items/_order_item.html.erb]: use `.nil?`

Comment (becousae) [inline app/views/order_items/_order_item.html.erb]: `.nil?` doesn't return true for 0. `zero?` does.

I don't think you need a ternary operator here. `<%= ' out-of-stock' if product.stock.zero? %>` should work too.

Comment (Silox) [inline app/views/order_items/_order_item.html.erb]: Right, english.



On 8 Jul 2015 at 14:43:54, benji (notifications@github.com) wrote:

In app/views/order_items/_order_item.html.erb:

> @@ -1,5 +1,5 @@
>  <div class="col-md-3 form_products">
> -  <div class="thumbnail ">
> +  <div class="thumbnail<%= product.stock ==0 ? ' out-of-stock' : '' %>">
.nil? doesn't return true for 0. zero? does.

I don't think you need a ternary operator here. <%= ' out-of-stock' if product.stock.zero? %> should work too.

**Pull request merged**